code/modules/flufftext/Hallucination.dm

```diff
@@ -976,6 +978,59 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	target.clear_alert(alert_type, clear_override = TRUE)
 	qdel(src)
 
+/datum/hallucination/fake_alert/proc/cleanup()
+	target.clear_alert(alert_type, clear_override = TRUE)
+	qdel(src)
+
+///Causes the target to see incorrect health damages on the healthdoll
+/datum/hallucination/fake_health_doll
+	var/timer_id = null
+
+///Creates a specified doll hallucination, or picks one randomly
+/datum/hallucination/fake_health_doll/New(mob/living/carbon/human/human_mob, forced = TRUE, specific_limb, severity, duration = 500)
+	. = ..()
+	if(!specific_limb)
+		specific_limb = pick(list(SCREWYDOLL_HEAD, SCREWYDOLL_CHEST, SCREWYDOLL_L_ARM, SCREWYDOLL_R_ARM, SCREWYDOLL_L_LEG, SCREWYDOLL_R_LEG))
```

Comment:
I don't really understand why there has to be defines for this, they're code typepaths

the list should probably be static anyway

Reply:
done in a8298c5